src/solvespace.cpp

```diff
@@ -70,6 +70,8 @@ void SolveSpaceUI::Init() {
     drawBackFaces = CnfThawBool(true, "DrawBackFaces");
     // Check that contours are closed and not self-intersecting
     checkClosedContour = CnfThawBool(true, "CheckClosedContour");
+    // Use turntable mouse navigation
+    checkTurntableNav = CnfThawBool(false, "CheckTurntableNav");
```

Comment:
"check" in "checkClosedContour" doesn't mean "checkbox", it's a part of the option name